website/static/js/pages/home-page.js

```diff
@@ -6,6 +6,7 @@
 
 var $ = require('jquery');
 var m = require('mithril');
+var $osf = require('js/osfHelpers');
```

Comment:
This file already requires `js/osfHelpers` a few lines up, so this can be removed.